src/js/utils/StringUtil.js

```diff
@@ -215,6 +215,16 @@ const StringUtil = {
     }, []);
 
     return this.idToTitle(splitID, splitBy, replace, removeConsecutive);
+  },
+
+  lrtrim(str, trimStr = " ") {
```

Comment:
Or we could also run .trim() at the end of the statement and leave the trimStr with no default value?